client-js/app/elements/labrad-nodes.html

```diff
@@ -73,10 +73,10 @@
       top: 5px;
     }
     paper-icon-button.autostart {
-      color: #75787B;
+      color: #85888B;
```

Comment:
Ok, reverted. I found this and the green too dark for my taste, but it's just that, taste :-)